plugins/services/src/js/service-configuration/ServiceStorageConfigSection.js

```diff
@@ -31,160 +28,140 @@ class ServiceStorageConfigSection extends ServiceConfigBaseSectionDisplay {
-                isHostVolume: false
-              };
-
-              volume.containerPath = appVolume.containerPath;
-              volume.mode = appVolume.mode;
-
-              if (appVolume.persistent != null) {
-                volume.size = appVolume.persistent.size;
-                volume.type.push("Persistent", "Local");
-              } else if (appVolume.external != null) {
-                volume.size = appVolume.external.size;
-                volume.type.push("External");
-              } else {
-                volume.isHostVolume = true;
-                volume.type.push("Host", "Volume");
-              }
-
-              if (appVolume.external != null) {
-                volume.name = appVolume.external.name;
-              }
-
-              if (appVolume.hostPath != null) {
-                // Set this flag to true so that we render the hostPath column.
-                shouldDisplayHostPath = true;
-              }
-
-              volume.hostPath = getDisplayValue(
-                appVolume.hostPath,
-                !volume.isHostVolume
-              );
-
-              return volume;
-            });
-
-            if (shouldDisplayHostPath) {
-              columns.push({
-                heading: getColumnHeadingFn("Host Path"),
-                prop: "hostPath",
-                className: getColumnClassNameFn(),
-                sortable: true
-              });
-            }
-
-            if (onEditClick) {
-              columns.push({
-                heading() {
-                  return null;
-                },
-                className: "configuration-map-action",
-                prop: "edit",
-                render() {
-                  return (
-                    <a
-                      className="button button-link flush table-display-on-row-hover"
-                      onClick={onEditClick.bind(null, "volumes")}
-                    >
-                      Edit
-                    </a>
-                  );
-                }
-              });
-            }
-
-            return (
-              <Table
-                key="service-volumes"
-                className="table table-flush table-borderless-outer table-borderless-inner-columns vertical-align-top table-break-word table-fixed-layout flush-bottom"
-                columns={columns}
-                data={volumesData}
-              />
-            );
-          }
         }
       ]
     };
+
+    const volumesConfig = volumes.map((volume, index) => {
+      if (volume.persistent != null && volume.persistent.profileName != null) {
+        // DSS
+        return [
+          {
+            heading: VolumeDefinitions.DSS.name,
+            headingLevel: 2
+          },
+          {
+            key: `container.volumes.${index}.persistent.profileName`,
+            label: "PROFILE NAME"
+          },
+          {
+            key: `container.volumes.${index}.containerPath`,
+            label: "CONTAINER PATH"
```

Comment:
@weblancaster Can you change these labels so they're not uppercase. To be consistent with the rest of the review and config view.

Reply:
Sounds good. I saw the prototype displaying all caps and forgot to follow what we already have instead which is not all caps. Thanks.